winlogbeat/sys/eventlogging/zsyscall_windows.go

```diff
@@ -21,7 +21,7 @@ var (
 	procGetOldestEventLogRecord    = modadvapi32.NewProc("GetOldestEventLogRecord")
 )
 
-func openEventLog(uncServerName *uint16, sourceName *uint16) (handle Handle, err error) {
+func _OpenEventLog(uncServerName *uint16, sourceName *uint16) (handle Handle, err error) {
```

Comment:
I wanted to retain the original method name as used in the Windows APIs, but not export the function outside of this golang package. Since only methods/fields that start with uppercase values are exported, adding an `_` (underscore) prevents it from being exported.